app/components/project-recommendation-new.js

```diff
@@ -0,0 +1,126 @@
+import { action } from '@ember/object';
+
+// Creates a new Borough Peesident Recommendation associated with one or more Actions.
+// Borough President Recommendations do not contain voting attributes.
+function createBoroughPresidentRecForActions(thisRec) {
+  const newRec = thisRec.store.createRecord('borough-president-recommendation', {
+    // general
+    user: thisRec.user,
+    // assume all actions for now
+    // TODO: filter actions by user's selection
+    actions: thisRec.project.get('actions'),
+    recommendation: 'Approved with Modifications/Conditions',
+    consideration: 'My awesome memo.',
+    voteLocation: '123 Boston Ave, Brooklyn, NY 11111',
+    dateReceived: new Date('2015-12-12T03:24:00'),
+    dateVoted: new Date('2014-12-17T01:23:45'),
+    isActive: 'active',
+    status: 'Submitted',
+    docketDescription: 'My awesome docket description.',
+  });
+
+  newRec.save();
+}
+
+function createCommunityBoardRecForActions(thisRec) {
+  const newRec = thisRec.store.createRecord('community-board-recommendation', {
+    // general
+    user: thisRec.user,
+    // assume all actions for now
+    // TODO: filter actions by user's selection
+    actions: thisRec.project.get('actions'),
+    recommendation: 'Approved with Modifications/Conditions',
+    consideration: 'My awesome memo.',
+    voteLocation: '123 Boston Ave, Brooklyn, NY 11111',
+    dateReceived: new Date('2015-12-12T03:24:00'),
+    dateVoted: new Date('2014-12-17T01:23:45'),
+    isActive: 'active',
+    status: 'Submitted',
+    docketDescription: 'My awesome docket description.',
+    // Community-Board specific
+    votesInFavor: 77,
+    votesAgainst: 22,
+    votesAbstain: 25,
+    totalBoardMembers: 14,
+    didQuorumExist: false,
+  });
+
+  newRec.save();
+}
+
+function createBoroughBoardRecForActions(thisRec) {
+  const newRec = thisRec.store.createRecord('borough-board-recommendation', {
+    // general
+    user: thisRec.user,
+    // assume all actions for now
+    // TODO: filter actions by user's selection
+    actions: thisRec.project.get('actions'),
+    recommendation: 'Approved with Modifications/Conditions',
+    consideration: 'My awesome memo.',
+    voteLocation: '123 Boston Ave, Brooklyn, NY 11111',
+    dateReceived: new Date('2015-21-12T03:24:00'),
+    dateVoted: new Date('2014-12-17T01:23:45'),
+    isActive: 'active',
+    status: 'Submitted',
+    docketDescription: 'My awesome docket description.',
+    // Community-Board specific
+    votesInFavor: 77,
+    votesAgainst: 22,
+    votesAbstain: 25,
+    totalBoardMembers: 14,
+    didQuorumExist: false,
+  });
+
+  newRec.save();
+}
+
+// Creates a new Recommendation based on a combination of
+// user inputted Recommendation attributes
+// and associates it to a given User, Project, and set of selected Actions
+// TODO: Hook up this component to user inputted data.
+export default class ProjectRecommendationNewComponent extends Component {
```

Comment:
This component has a lot of static data in it... was setting it up with a mirage-backed workflow too time consuming?

Reply:
Wanted to do this in the PR/Issue 533 for actually creating a Recc form. See comment above